headers/headers.go

```diff
@@ -10,16 +10,47 @@ import (
 	"github.com/ONSdigital/log.go/log"
 )
 
-var (
-	// CollectionIDKey is the name used for a collection ID http request header
-	CollectionIDKey = "Collection-Id"
+const (
+	// CollectionIDHeader is the name used for a collection ID http request header
+	CollectionIDHeader = "Collection-Id"
```

Comment:
Do we think that the header names should be private? That will help prevent people accidentally duplicating these functions.

Reply:
If there isn't a benefit for them being public then I would think private. I think keep accessors as limiting as they can be is a good practice to follow, nothing public that doesn't need to be etc.

So IMO Yes.